test/e2e/action_test.go

```diff
@@ -319,6 +395,40 @@ func getTypeInstanceInputForUpdate() *hublocalgraphql.CreateTypeInstanceInput {
 	}
 }
 
+func getTypeInstanceHelmStorage() *hublocalgraphql.CreateTypeInstanceInput {
```

Comment:
Maybe something like:
```suggestion
func fixHelmStorageTypeInstanceCreateInput() *hublocalgraphql.CreateTypeInstanceInput {
```